src/utils/contract.ts

```diff
@@ -29,11 +29,29 @@ export default class Contract {
     if (isNullAddress(response))
       throw new ResolutionError(ResolutionErrorCode.RecordNotFound, {
         recordName: method,
-        domain: args[0],
+        domain: args[args.length - 1] as string,
       });
     return this.coder.decodeFunctionResult(method, response)[0]
   }
 
+  async fetchLogs(eventName: string, tokenId: string): Promise<EventData[]> {
+    const topic = this.coder.getEventTopic(eventName);
+    const params = [
+      {
+        fromBlock: "0x960844",
+        toBlock: "latest",
```

Comment:
nitpick: single quotes

P.S. we do need to automate prettier runs after this PR...